distro/configs/openfn/openfn.env

```diff
@@ -0,0 +1,191 @@
+
+# What ip:port should be published back to the Docker host for the app server?
+# If you're using Docker Toolbox or a custom VM you can't use 127.0.0.1. This
+# is being overwritten in dev to be compatible with more dev environments.
+#
+# If you have a port conflict because something else is using 4000 then you
+# can either stop that process or change 4000 to be something else.
+#
+# Use the default in production to avoid having it directly accessible to
+# the internet without assistance from a cloud based firewall.
+#LIGHTNING_EXTERNAL_PORT=127.0.0.1:4000
+LIGHTNING_EXTERNAL_PORT=4000
+
+# What volume path should be used? In dev we want to volume mount everything
+# so we can develop our code without rebuilding our Docker images.
+# Note that by mounting the whole project into the container, you will need to
+# follow the Contributing steps in the README.
+#LIGHTNING_VOLUME=.:/app
+
+# What CPU and memory constraints will be added to your services? When left at
+# 0, they will happily use as much as needed.
+#DOCKER_POSTGRES_CPUS=0
+#DOCKER_POSTGRES_MEMORY=0
+#DOCKER_WEB_CPUS=0
+#DOCKER_WEB_MEMORY=0
+
+# Give this variable the value of true if you want the system to create a sample project for a new registered user
+INIT_PROJECT_FOR_NEW_USER=false
+
+# If not provided, PURGE_DELETED_AFTER_DAYS defaults to 7. Set to 0 to never purge deleted records.
+PURGE_DELETED_AFTER_DAYS=7
+
+# To use https://plausible.io/ analytics, provide the SRC for your script and
+# your data-domain below.
+# PLAUSIBLE_SRC=https://plausible.io/js/script.js
+# PLAUSIBLE_DATA_DOMAIN=openfn.org
+
+# If you wish to enable PromEx-driven Prometheus/Grafana monitoring use the following:
+# PROMEX_ENABLED=true
+# PROMEX_GRAFANA_HOST=http://localhost:3000
+# PROMEX_GRAFANA_USER=admin
+# PROMEX_GRAFANA_PASSWORD=admin
+# PROMEX_UPLOAD_GRAFANA_DASHBOARDS_ON_START=true
+# PROMEX_DATASOURCE_ID=promex
+# PROMEX_METRICS_ENDPOINT_AUTHORIZATION_REQUIRED=yes
+# PROMEX_METRICS_ENDPOINT_TOKEN=foobar
+# PROMEX_ENDPOINT_SCHEME=http
+
+# The length of time a Run must remain in the `available` state before it is
+# considered `stalled`.
+# METRICS_STALLED_RUN_THRESHOLD_SECONDS=300
+
+# The maximum age of a Run that will be considered when measuring Run performance.
+# METRICS_RUN_PERFORMANCE_AGE_SECONDS=120
+
+# To disable the reporting of anonymised metrics to the OpenFn Usage tracker, set
+# USAGE_TRACKING_ENABLED to `false`.
+# USAGE_TRACKING_ENABLED=false
+
+# To submit cleartext UUIDs to the usage tracker (default: hashed_only),
+# set USAGE_TRACKING_UUIDS=cleartext.
+# USAGE_TRACKING_UUIDS=hashed_only
+
+# By default, impact tracking metrics will be reported to https://impact.openfn.org. Use the
+# below if you wish to change that.
+# USAGE_TRACKER_HOST=https://impact.openfn.org
+
+# Solving "ArgumentError" https://community.openfn.org/t/lightning-prebuilt-images-throw-no-matching-manifest-for-linux-arm64-v8-in-the-manifest-list-entries/465/15?u=jnsereko
+ERL_FLAGS="+JPperf true"
```

Comment:
@jnsereko , i think this shouldn't be here. (assuming you're not deploying this on a mac m3)

Reply:
this is true.  We shall remove this in production

Reply:
@taylordowns2000 which approach should we use to manage what should be changed in production vs development while streamline the whole process so that we do not need to change alot on of things to make it work either locally or in cloud

Reply:
In addition @taylordowns2000, this seamed to work on ubuntu 22 with Gitpod

Reply:
good question. frankly if you're doing erlang/elixir development and you're running the whole thing using these pre-baked dockerhub images, i'd say run everything in production mode. but i can also ask for opinions from some of the other folks on the team

Reply:
I have disabled the `ERL_FLAGS` 
@pirupius you might need to enable this when testing locally